daemon/graphdriver/driver.go

```diff
@@ -50,10 +57,10 @@ type ProtoDriver interface {
 	String() string
 	// CreateReadWrite creates a new, empty filesystem layer that is ready
 	// to be used as the storage for a container.
-	CreateReadWrite(id, parent, mountLabel string, storageOpt map[string]string) error
+	CreateReadWrite(id, parent string, opts *CreateOpts) error
 	// Create creates a new, empty, filesystem layer with the
 	// specified id and parent and mountLabel. Parent and mountLabel may be "".
```

Comment:
Update comment to mention create options instead of mount label and mention that the create options may be nil.